docs/getting-started/install.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+title: Getting Started
+subtitle: An overview of Athena, including how to download and use it, some basic templates and examples, and more.
+---
+
+# Installation
+
+Installing the Athena Framework is as simple as adding the CSS and JS files to your project.
+
+<hr />
+
+## Minified files
+
+After <a href="{{ "/getting-started/download/" | prepend: site.baseurl }}">downloading and extracting the Athena-Framework,</a> add the CSS, JS and Font files to your project's static assets folder.
+
+Add Athena's CSS in the <code>&lt;head&gt;</code> of your project, before any additional CSS files:
+
+<pre><code>&lt;link rel="stylesheet" href="/css/framework.min.css"&gt;
+</code></pre>
+
+Athena's Javascript should be added before the closing <code>&lt;/body&gt;</code> tag. <a href="https://jquery.com/">jQuery</a> and <a href="http://tether.io/">Tether</a> are required dependencies for Athena, so be sure to include those before the Athena JS.
+
+<pre><code>&lt;script type="text/javascript" src="https://code.jquery.com/jquery-3.1.1.slim.min.js"&gt;&lt;/script&gt;
+&lt;script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"&gt;&lt;/script&gt;
+&lt;script type="text/javascript" src="/js/framework.min.js"&gt;&lt;/script&gt;
+</code></pre>
+
```

Comment:
Adding instructions for installing with a cloud.typography css key (and the base set of fonts necessary to get up and running) would be helpful for folks that want to use Gotham/Archer/Knockout/Chronicle.  We might also want to tell folks here not to use the main site's css key, just in case.  We could probably repurpose most of what's in this commit (installation.html): https://github.com/UCF/Athena-Framework/commit/fcc4bb20f8458fd38260d3760b4ce3a2849be40b